decompiler/util/TP_Type.h

```diff
@@ -7,6 +7,10 @@
 #include "common/common_types.h"
 #include "decompiler/Disasm/Register.h"
 
+#ifdef __APPLE__
+#include <unordered_map>
```

Comment:
same here, I should have had this in the first place. you can remove the `__APPLE__` check

Reply:
Perfect! I wasn't sure on the proper implementation for this